NN_chatCmd.js

```diff
@@ -1,21 +1,164 @@
+nerthus.chatCmd = {}
+nerthus.chatCmd.map = {}
+nerthus.chatCmd.public_map = {}
 
-nerthus.chatCmd = {};
-nerthus.chatCmd.map = {};
-nerthus.chatCmd.public_map = {};
+nerthus.chatCmd.cards = {}
+nerthus.chatCmd.cards.currentDecks = {}
+nerthus.chatCmd.cards.currentDecks[52] = {}
+nerthus.chatCmd.cards.currentDecks[54] = {}
+nerthus.chatCmd.cards.saveCardsForSession = function (data, ts)
+{
+    sessionStorage.setItem("nerthus_cards_" + ts, JSON.stringify(data))
+}
+nerthus.chatCmd.cards.loadCardsFromSession = function (ts)
+{
+    const cards = sessionStorage.getItem("nerthus_cards_" + ts)
+    if (cards)
+        return JSON.parse(cards)
+}
+nerthus.chatCmd.cards.pseudoRandom = function (seed)
+{
+    seed = (seed * 9301 + 49297) % 233280
+    return seed / 233280
+}
+nerthus.chatCmd.cards.getCard = function (deckId, ts, deck_type)
```

Comment:
strasznie długie są te funkcje z kartami, można z nich wyekstrahować funkcje pomocnicze, np. draw card które zwróci odr azu nowy obiekt z id, kolorem i nazwą

Reply:
No i jest funkcja getCard od tego, by jedną kartę wziąć i zwrócić obiekt id/kolor/nazwa. Nie wiem czy jeszcze większe rozdrabnianie tego jest potrzebne.

Reply:
Można kilka kart naraz dobierać, stąd są skomplikowane. Jakbym miał przenieść to by to tylko zmieniło miejsce, nadal by były długie.

Reply:
to może jakaś funkcje która konwertuje id na value i color, bo to zabiera tu sporo miejsca

Reply:
mogę dodać takową